docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -57,7 +84,7 @@ Use the following to access the SSH service on the worker nodes:
 |----------|-----------|------------|-------------------------|---------------------------|
 | TCP      | Inbound   | 22         | SSHD server             | SSH clients               |
 
-## Bare Metal provider
+##### Bare Metal provider
```

Comment:
```suggestion
### Bare Metal provider
```
I suggested changing the heading above to ##. If you agree, then this should be ###.